windows/ReactNativeWebView2/ReactWebView2.cpp

```diff
@@ -0,0 +1,126 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "JSValueXaml.h"
+#include "ReactWebView2.h"
+#include "ReactWebView2.g.cpp"
+#include <winrt/Windows.Foundation.Metadata.h>
+#include <optional>
+
+namespace winrt {
+    using namespace Microsoft::ReactNative;
+    using namespace Windows::Data::Json;
+    using namespace Windows::Foundation;
+    using namespace Windows::UI;
+    using namespace Windows::UI::Popups;
```

Comment:
some of these usings and corresponding includes don't seem to be needed, remove where you can

Reply:
These were taken from the original ReactWebView source, but I can investigate.